rules/imports.js

```diff
@@ -0,0 +1,159 @@
+module.exports = {
+  plugins: ['import'],
+
+  settings: {
+    'import/resolver': {
+      node: {
+        extensions: ['.js', '.json'],
+      },
+    },
+    'import/extensions': ['.js'],
+    'import/core-modules': [],
+    'import/ignore': ['node_modules', '\\.(scss|css|less|hbs|svg|json)$'],
```

Comment:
Why?

Reply:
I suppose AirBnb did this mainly for webpack loaders, but let's try removing it for now and we can add later if needed.